src/main/docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM openjdk:17-oracle
+RUN groupadd -r phs && useradd -r -g phs phs
+USER phs:phs
+ARG DEPENDENCY=./target/dependency
+COPY ${DEPENDENCY}/BOOT-INF/lib /app/lib
+COPY ${DEPENDENCY}/META-INF /app/META-INF
+COPY ${DEPENDENCY}/BOOT-INF/classes /app
+ENTRYPOINT ["java","-cp","./target/dependency/BOOT-INF/lib:/app/lib/*","io.jenkins.pluginhealth.scoring.PluginHealthScoring"]
```

Comment:
I'm currently trying to understand what to put before the ":/app/lib/*" in the third element inside this array. I'll do some googling on this.

Reply:
Please see https://github.com/jenkins-infra/plugin-health-scoring/pull/7/commits/0486ea82807c5145f8ba09896198b560f5716c5e

You still need to `mvn package` before you can `docker image build` it's working now.